src/frontend/Sider/Behandling/Venstremeny/Oppsummering/NyeOpplysningerMetadata.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+
+import { BodyShort, HStack, VStack } from '@navikt/ds-react';
+
+import {
+    nyeOpplysningerEndringTilTekst,
+    NyeOpplysningerMetadata,
+} from '../../../../typer/behandling/nyeOpplysningerMetadata';
+
+const NyeOpplysningerMetadata: React.FC<{
+    nyeOpplysningerMetadata: NyeOpplysningerMetadata;
+}> = ({ nyeOpplysningerMetadata }) => {
+    const endringerVisning = (() => {
+        if (nyeOpplysningerMetadata.endringer.length === 0) {
+            return <BodyShort size="small">-</BodyShort>;
+        }
+        if (nyeOpplysningerMetadata.endringer.length === 1) {
+            return (
+                <BodyShort size="small">
+                    {nyeOpplysningerEndringTilTekst[nyeOpplysningerMetadata.endringer[0]]}
+                </BodyShort>
+            );
+        }
+
+        return nyeOpplysningerMetadata.endringer.map((endring, index) => {
+            const isLast = index === nyeOpplysningerMetadata.endringer.length - 1;
+            return (
+                <BodyShort key={index} size="small">
+                    {nyeOpplysningerEndringTilTekst[endring]}
+                    {!isLast && ','}
+                </BodyShort>
+            );
+        });
+    })();
+
+    return (
+        <VStack>
+            <HStack gap={'1'}>
+                <BodyShort size={'small'} weight={'semibold'}>
+                    Kilde:
+                </BodyShort>
+                <BodyShort size={'small'}>{nyeOpplysningerMetadata.kilde}</BodyShort>
```

Comment:
Burde denne ha en {kildeTilTekst[nyeOpplysningerMetadata.kilde]} ? Så får man ikke uppercase for alle typene 👼